libfreerdp/crypto/privatekey.c

```diff
@@ -422,6 +422,8 @@ BOOL freerdp_key_generate(rdpPrivateKey* key, size_t key_length)
 
 char* freerdp_key_get_param(const rdpPrivateKey* key, enum FREERDP_KEY_PARAM param, size_t* plength)
 {
+	BYTE* buf = NULL;
```

Comment:
This function still might return an "uninitialized" (zeroed) buffer (i.e. if `BN_bn2bin` fails)